renderer-protocol/proto/decentraland/renderer/renderer_services/restricted_actions.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+package decentraland.renderer.renderer_services;
+
+message OpenExternalUrlRequest {
+  string url = 1;
+  int32 scene_number = 2;
+}
+
+message OpenNftDialogRequest {
+  string urn = 1;
+}
+
+message SuccessResponse {
```

Comment:
Are we not saying the same thing twice? Maybe renaming to `Response` would be better because it can be a non-successful response. Right?